.github/workflows/docker-update-images.yaml

```diff
@@ -0,0 +1,26 @@
+# Continuously build Rosetta
+name: Docker Update Images
+
+on:
```

Comment:
Add a "pull request" that matches this file and the python file path

That way your PR will actually run and test this workflow

Reply:
Also you'll wanna add a workflow_dispatch so that it can be triggered manually as well